src/microbe_stage/MicrobeStage.cs

```diff
@@ -367,6 +367,15 @@ public void MoveToMulticellular()
             return;
         }
 
+        // Log becoming multicellular in the timeline
+        GameWorld.LogEvent(
+            new LocalizedString("TIMELINE_PLAYER_BECAME_MULTICELLULAR", Player.Species.FormattedName),
```

Comment:
In the future becoming multicellular is not exclusive to the player, so I think the translation key should be "TIMELINE_SPECIES_BECAME_MULTICELLULAR" (and the text needs to be adjusted as well if it makes explicit reference to the player).